<def f='linux/arch/x86/include/asm/trace/exceptions.h' l='14' macro='1'/>
<use f='linux/arch/x86/include/asm/trace/exceptions.h' l='14' macro='1' c='trace_raw_output_x86_exceptions'/>
<use f='linux/arch/x86/include/asm/trace/exceptions.h' l='14' macro='1' c='trace_event_define_fields_x86_exceptions'/>
<use f='linux/arch/x86/include/asm/trace/exceptions.h' l='14' macro='1' c='trace_event_get_offsets_x86_exceptions'/>
<use f='linux/arch/x86/include/asm/trace/exceptions.h' l='14' macro='1' c='trace_event_raw_event_x86_exceptions'/>
<use f='linux/arch/x86/include/asm/trace/exceptions.h' l='14' macro='1' c='perf_trace_x86_exceptions'/>
<size>32</size>
<mbr r='trace_event_raw_x86_exceptions::ent' o='0' t='struct trace_entry'/>
<mbr r='trace_event_raw_x86_exceptions::address' o='64' t='unsigned long'/>
<mbr r='trace_event_raw_x86_exceptions::ip' o='128' t='unsigned long'/>
<mbr r='trace_event_raw_x86_exceptions::error_code' o='192' t='unsigned long'/>
<mbr r='trace_event_raw_x86_exceptions::__data' o='256' t='char [0]'/>
